h/static/scripts/root-thread.js

```diff
@@ -63,6 +63,14 @@ function RootThread($rootScope, annotationUI, searchFilter, viewFilter) {
       };
     }
 
+    var threadFilterFn = function (thread) {
+      if(state.selectedTab === 'annotation') {
+        return (metadata.isTypeAnnotation(thread.annotation) || metadata.isTypeReply(thread.annotation));
```

Comment:
`isTypeReply()` will always be false here because only top-level annotations (ie. not replies) are being filtered using `threadFilterFn`.